fearless/Common/Services/GitHubPhishingService/GitHubPhishingAPIService.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+import RobinHood
+
+class GitHubPhishingAPIService: ApplicationServiceProtocol {
+    lazy var endPoint: String = { return "https://polkadot.js.org/phishing/address.json" }()
+
+    private var operation: BaseOperation<[PhishingItem]>
+    private var logger: Logger
+    private var storage: CoreDataRepository<PhishingItem, CDPhishingItem>
+
+    init(operation: BaseOperation<[PhishingItem]>,
```

Comment:
What if we will need to execute an opreration multiple time. I guess we need to pass an operation factory here